spec/support/evss/mock_letters_response.yml

```diff
@@ -39,11 +39,11 @@
         has_survivors_pension_award: false
         monthly_award_amount: 123.0
         service_connected_percentage: 2
-      has_adapted_housing: false
-      has_chapter35_eligibility: true
-      has_death_result_of_disability: false
-      has_individual_unemployability_granted: false
-      has_special_monthly_compensation: false
+        has_adapted_housing: false
```

Comment:
Typically, you'd be correct, but this looks to be for the rspecs.  @kreek checked this file in to begin with.  I think it's an exception.